src/views/ApartmentApp/components/StudentApplication/components/InstructionsCard/index.js

```diff
@@ -22,13 +23,10 @@ const InstructionsCard = () => {
   const [thisYear, setThisYear] = useState();
 
   useEffect(() => {
-    const getYear = () => setThisYear(new Date().getFullYear());
-
-    getYear();
-
-    // let selectionDate = housing.getApartmentSelectionDate();
-    let selectionDate = 'Apr. 27'; // API endpoint for this is planned for the future
-    setApartmentSelectionDate(selectionDate);
+    setThisYear(new Date().getFullYear());
+    housing
+      .getApartmentSelectionDate()
+      .then((selectionDate) => setApartmentSelectionDate(selectionDate));
```

Comment:
Fun fact about `Promise.then` and other functions that expect a callback - when you simply want to call a function on the parameters, you can pass that function directly, rather than passing an arrow function that does the same thing:
```suggestion
      .then(setApartmentSelectionDate);
```
In this case, I think it actually obscures the code more than simplifying it, but it's a neat thing to keep in mind.

It can be especially helpful defining the `onClick` prop of components. For example, rather than having an input with
`onClick={(event) => setInputValue(event.target.value)}`
you can simply do
`onClick={setInputValue}`

Reply:
I would want to double check the `.then` documentation, because I know that there can be unexpected behavior if there are any optional secondary or tertiary parameters involved. 
I suspect this will change again in the future when we actually make the endpoint on the API side
But I agree in general, and in this case the simplified form might make it look a little more confusing to read, so i'll leave it as-is for now